home-page/themes.js

```diff
@@ -126,6 +126,36 @@ let summerday = {
   "widget-background": deepcoral,
   "section-header-color": brick,
 };
+/// Brian Loveless Pumpkin Spice fall
```

Comment:
I love this theme 😄
Also:

<img width="290" alt="Screen Shot 2020-08-26 at 7 38 00 AM" src="https://user-images.githubusercontent.com/47202564/91310506-31bbd180-e76f-11ea-8367-95eb990c1bb3.png">